### ISSUE TITLE
Accessability Violation : The ARIA attributes "aria-label" are not valid for the element <span> with implicit ARIA role "generic"

### ISSUE THREAD

**Issue by sant-santhiya** [carbon-design-system/carbon-components-angular#2529]
## Detailed description

> Describe in detail the issue you're having.
The ARIA attributes "aria-label" are not valid for the element <span> with implicit ARIA role "generic"

> Is this a feature request (new component, new icon), a bug, or a general issue?
Bug

> Is this issue related to a specific component?
`ibm-date-picker -- > <span tabindex="-1" aria-label="date" class="flatpickr-day prevMonthDay bx--date-picker__day">`

> What did you expect to happen? What happened instead? What would you like to see changed?
Need to remove aria-label in span tag of` <ibm-date-picker>`

> What browser are you working in?
Mozila Firefox

> What version of the Carbon Design System are you using?
```
"carbon-components": "^10.56.0",
"carbon-components-angular": "^4.59.8",
```

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?
within one week

## Steps to reproduce the issue

Run the IBM Accessibility Checker tool on the component by opening the calendar. [components-date-picker--single](https://angular.carbondesignsystem.com/?path=/story/components-date-picker--single)

## Additional information
![image](https://user-images.githubusercontent.com/104894503/234175224-1b9e643d-5202-41fb-b42c-487ded64aee0.png)

##Element location
<span
    tabindex="-1"
    aria-label="March 26, 2023"
    class="flatpickr-day prevMonthDay bx--date-picker__day">
